### ISSUE TITLE
One extra session is created when running tests on cloud browser providers.

### ISSUE THREAD

**Issue by Ankit098** [smooth-code/jest-puppeteer#452]
## 🐛 Bug Report

When using the connect method to run tests, one additional blank session is getting created per run on cloud browser providers, for example, 5 sessions for a 4 test suite. When openBrowser method in [global.js](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/src/global.js) gets called during setup, it creates a session which is not used to run any tests. 

## To Reproduce

Run a basic test on BrowserStack, with say 2 tests. 
[Sample](https://www.browserstack.com/docs/automate/puppeteer/jest)

## Expected behavior

Ideally 2 sessions should be seen on the BrowserStack dashboard but currently you'd see three sessions two corresponding to the two tests but an extra third one as a blank session.

## Link to repl or repo (highly encouraged)

[Repo](https://github.com/browserstack/puppeteer-browserstack)
[Docs](https://www.browserstack.com/docs/automate/puppeteer/jest)

Issues without a reproduction link are likely to stall.

## Run `npx envinfo --system --binaries --npmPackages expect-puppeteer,jest-dev-server,jest-environment-puppeteer,jest-puppeteer,spawnd --markdown --clipboard`

Paste the results here:

```bash
## System:
 - OS: macOS 11.1
 - CPU: (8) x64 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
 - Memory: 229.95 MB / 8.00 GB
 - Shell: 5.8 - /bin/zsh
## Binaries:
 - Node: 12.13.0 - ~/.nvm/versions/node/v12.13.0/bin/node
 - Yarn: 1.22.10 - ~/.nvm/versions/node/v12.13.0/bin/yarn
 - npm: 6.12.0 - ~/.nvm/versions/node/v12.13.0/bin/npm
## npmPackages:
 - jest-puppeteer: 6.0.0 => 6.0.0
```

**Comment by UziTech** [comment]
How do you propose this is fixed? I don't think there is anything we can change in jest-puppeteer to fix this. This seems like an issue with browser stack.

**Comment by Ankit098** [comment]
Could have a flag/config option to avoid the connect call inside `global.js` and [here](https://github.com/smooth-code/jest-puppeteer/blob/8584363304b3d1f94cc0b5f11d75b5dc9b7abe60/packages/jest-environment-puppeteer/src/global.js#L35) use the `wsEndPoint` provided in config. This would use the connect method [here](https://github.com/smooth-code/jest-puppeteer/blob/8584363304b3d1f94cc0b5f11d75b5dc9b7abe60/packages/jest-environment-puppeteer/src/PuppeteerEnvironment.js#L121) to run the tests.

**Comment by UziTech** [comment]
The connect call is required to get the browser variable for the wsEndPoint.

**Comment by UziTech** [comment]
should be fixed in v6.0.3
